sdk/appconfiguration/app-configuration/test/README.md

```diff
@@ -0,0 +1,62 @@
+# Testing
+
+To run our tests, first do a full update of the project by running `rush update`, then rebuild either this package, with `rush build -t @azure/app-configuration` or all packages with `rush build`, then make sure to build the tests with `rush build:test`. After that, you can either run our unit tests, which won't reach to live services, and our integration tests, which will indeed require existing, reachable resources.
+
+## Run all tests
```

Comment:
Assuming we are moving to a `Testing` section in the contributing.md file, I would recommend the below structure

```
### Testing

Run `rushx test` from the root of the package that you want to run tests for. 
This will run tests both in node.js and browser. To target them individually, 
you can run `rushx test:node` and `rushx test:browser`

#### Recorded tests

By default, the npm scripts `test`, `test:node` or `test:browser` run recorded tests.
<<Add the content that you now have under "Unit Tests">>

#### Live tests

By default, the npm scripts `test`, `test:node` or `test:browser` run recorded tests.
To run live tests, set the environment variable TEST_MODE to "live" and ensure you have the right 
Azure resources to test against.
<< Add the content that you now have under "integration tests">>

#### Debugging tests using Visual Studio Code

```

I would skip the below sections you have now. These are generic instructions on how to use mocha
- Run all tests in a single test suite
- Run a single test

Reply:
Thank you so much! This really helps!